### DISCUSSION TITLE
Will MIDI get some love?

### DISCUSSION THREAD

**Discussion by ghost** [tenacityteam/tenacity-legacy#452] [Ideas]
Hey!

Are there any thoughts about getting more MIDI stuff included? Like a piano roll? :-)

Thanks!

**Comment by caughtquick** [comment]
I'm daily certain MiDi doesn't work properly or at all with upstream PortAudio so we would be looking to see if we can get it working with upstream before focusing on any improvements to it. PRs are always welcome.

**Reply by ghost** [reply]
It makes sense. :-)
